soc/espressif/esp32c2/Kconfig

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2024 Espressif Systems (Shanghai) Co., Ltd.
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_ESP32C2
+	select RISCV
+	select RISCV_GP
+	select DYNAMIC_INTERRUPTS
+	select CLOCK_CONTROL
+	select PINCTRL
+	select RISCV_ISA_RV32I
+	select RISCV_ISA_EXT_A
+	select RISCV_ISA_EXT_M
+	select RISCV_ISA_EXT_C
+	select RISCV_ISA_EXT_ZICSR
+	select HAS_ESPRESSIF_HAL
+	select XIP if !MCUBOOT
+
+if SOC_SERIES_ESP32C2
+
+config ESP32_PHY_MAX_WIFI_TX_POWER
```

Comment:
This (and other) has been moved into `Kconfig.wifi`. Not sure if merged already.